scripts/fetch/site_stats.R

```diff
@@ -0,0 +1,28 @@
+fetchTimestamp.site_stats <- vizlab::alwaysCurrent
+
+#' @title Get the discharge quantiles for each dv gage
+#' 
+#' @param viz a vizlab object that depends on a vector of sites
+fetch.site_stats <- function(viz){
+  library(dataRetrieval)
+  
+  deps <- readDepends(viz)
+  checkRequired(deps, "sites")
+  sites <- deps[["sites"]]
+  
+  req_bks <- seq(1, length(sites),by=10)
+  stat_data <- data.frame()
+  for(i in req_bks) {
+    get_sites <- sites[i:(i+9)]
```

Comment:
I think the block chunk (here it is 10) should be a parameter, and this `(i+9)` should be `(i+blk_n-1)` so we don't have these two magic numbers floating around.